SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -14,30 +15,39 @@ public final class SwiftGenStoryboardEnumBuilder {
         
         class ParserDelegate : NSObject, NSXMLParserDelegate {
             var scenes = [Scene]()
+            var segues = [Scene]()
             var inScene = false
             var readyForFirstObject = false
+            var readyForConnections = false
             
             @objc func parser(parser: NSXMLParser, didStartElement elementName: String,
-                namespaceURI: String?, qualifiedName qName: String?,
-                attributes attributeDict: [String : String])
+              namespaceURI: String?, qualifiedName qName: String?,
+              attributes attributeDict: [String : String])
             {
-                
-                switch elementName {
-                case "scene":
-                    inScene = true
-                case "objects" where inScene:
-                    readyForFirstObject = true
-                case _ where readyForFirstObject:
-                    if let storyboardID = attributeDict["storyboardIdentifier"] {
-                        let customClass = attributeDict["customClass"]
-                        scenes.append(Scene(storyboardID, customClass))
-                    }
-                    readyForFirstObject = false
-                default:
-                    break
+              
+              switch elementName {
+              case "scene":
+                inScene = true
+              case "objects" where inScene:
+                readyForFirstObject = true
+              case _ where readyForFirstObject:
+                if let storyboardID = attributeDict["storyboardIdentifier"] {
+                  let customClass = attributeDict["customClass"]
+                  scenes.append(Scene(storyboardID, customClass))
                 }
+                readyForFirstObject = false
+              case "connections":
+                readyForConnections = true
+              case _ where readyForConnections:
```

Comment:
Yeah, good point. Probably don't do it in the storyboard actually shipped with the playground & code, but still worth checking in a storyboard created aside :wink: 

(That's why we need #2 to add tests :smile: )